### DISCUSSION TITLE
Question about CanadaSHM6 repository

### DISCUSSION THREAD

**Discussion by xjian257** [OpenDRR/opendrr#141] [General]
I have a question about CanadaSHM6 repository.  I can not find the repository at the OpenDRR. 
Could you please tell me where I could find the repository?

**Comment by tieganh** [comment]
Hi there @xjian257,

Unfortunately that repo is not publicly available yet. NRCan is working on getting all final approvals and documentation in place to be able to release this to the public, as soon as possible. Thanks for your patience.

Cheers,
Tiegan
